alerting/src/main/kotlin/com/amazon/opendistroforelasticsearch/alerting/resthandler/RestSearchMonitorAction.kt

```diff
@@ -62,6 +65,10 @@ class RestSearchMonitorAction : BaseRestHandler() {
 
     @Throws(IOException::class)
     override fun prepareRequest(request: RestRequest, client: NodeClient): RestChannelConsumer {
+        log.debug("${request.method()} ${AlertingPlugin.MONITOR_BASE_URI}/_search")
+
+        val index = request.param("index", SCHEDULED_JOBS_INDEX)
+
```

Comment:
How this change is related to `GET /destinations GET /alerts`?

Reply:
I had included it because it was needed for the security changes. I will remove this change since it is already included in ur PR.